### ISSUE TITLE
One more simple example for getting start guide

### ISSUE THREAD

**Issue by ryantibs** [cmu-delphi/epiprocess#39]
We should add more more example at the bottom of the [getting started guide](https://cmu-delphi.github.io/epiprocess/articles/epiprocess.html) (there's a "todo" at the bottom to mark the spot).

I'm thinking it'd be nice to have one more example just getting some data into `epi_df` format and then plotting it. Can be something from the `outbreaks` package, or something from the web somewhere (but *not* from `covidcast`). 

cc @rafaelcatoia @qpmnguyen In case you guys are interested.

**Comment by rafaelcatoia** [comment]
@ryantibs, I can do it. 

I tried to install the package as indicated in the [getting started guide](https://cmu-delphi.github.io/epiprocess/articles/epiprocess.html) but I get what seems to be a permission error related to `delphi-epidata-r` . I also tried to install `delphi-epidata-r` separately and got a different error:
  ```
Error in grepl(e, files, perl = TRUE, ignore.case = TRUE) : 
     invalid regular expression '^\Makefile$' 
```
Is that expected to happen and I should manually run the functions?

**Comment by ryantibs** [comment]
Thanks!

Re that error: my guess is that it's because of these two lines at the bottom of the DESCRIPTION file: 

```
Suggests:
    delphi.epidata
Remotes:
    github:cmu-delphi/delphi-epidata-r
```

I think that was added by @lcbrooks but it's possible there was a merge conflict somewhere that messed them up. So you could either fix the reference to the `delphi-epidata-r` repo (should be fine, since it's a public repo) in the DESCRIPTION, or simply remove these two lines from the DESCRIPTION, since at the moment we don't need the `delphi.epidata` package at all in any of the examples/vignettes. 

Either way, you could commit this change to your PR with the new example, and then when we merge it in, it'll fix main as well.

**Comment by rafaelcatoia** [comment]
Thanks, I'm running it locally and I'll let you know when I'm done

**Comment by rafaelcatoia** [comment]
@ryantibs, can you give me permission to push to the repo?

**Comment by ryantibs** [comment]
Sure, I just gave you write access. Just so you know, you should be able to create a fork and submit a PR even without write access.

And in any case you should always be creating a branch (or fork) and submitting a PR for new developments, in case that wasn't clear. Thanks!

**Comment by ryantibs** [comment]
Closed via #43.
